src/Nest/CommonOptions/Stats/RefreshStats.cs

```diff
@@ -13,5 +13,11 @@ public class RefreshStats
 
 		[DataMember(Name ="total_time_in_millis")]
 		public long TotalTimeInMilliseconds { get; set; }
+
+		[DataMember(Name ="external_total")]
+		public long ExternalTotal { get; set; }
```

Comment:
minor nitpick: XML doc to indicate that this is only valid for Elasticsearch 7.2.0+